andela/pippin-ah-frontend
# [Feature: #163788958] Footer component

#### What does this PR do?
- Create footer component

#### Description of Task to be completed?
- [x] install font awesom packages
- [x] refactor index.html file by removing fontawesom cdn 
- [x] create and style footer component 
- [x] add footer component to App component
- [x] write test for footer component [163788958]

#### How should this be manually tested?
- run `npm instal` and `npm run start:dev` to see the home page attached with the footer component
- run `npm run test:dev`  to see the test passing on local

#### What are the relevant pivotal tracker stories?
- [#163788958](https://www.pivotaltracker.com/story/show/163788958)

#### Screenshots (if appropriate)
<img width="1416" alt="screen shot 2019-02-07 at 6 59 50 pm" src="https://user-images.githubusercontent.com/32772323/52432400-9de1d080-2b0a-11e9-8121-152713e83bfb.png">
